web/src/tools/testing/recorder/build.sh

```diff
@@ -15,6 +15,8 @@ SUBPROJECT_NAME=tools/testing/recorder
 # This script runs from its own folder
 cd "$THIS_SCRIPT_PATH"
 
+BUNDLE_CMD="node $KEYMAN_ROOT/common/web/es-bundling/build/common-bundle.mjs"
```

Comment:
`BUNDLE_CMD` is already defined in `common.inc.sh`, so we wouldn't have to define it here.